tests/api/test_user.py

```diff
@@ -272,6 +272,91 @@ def test_add_unitadmin_user_with_unitpersonnel_permission_denied(client):
     assert invited_user is None
 
 
+def test_invite_user_expired_not_deleted(client):
+    """The invite token expires passed 168 hours (7 days) and every night at midnight a cronjob deletes the row in the db
+    However, if the token is expired and the cronjob hasn't exec yet (i.e the invit is still in the DB), users should be able to send a new invite
+    That replaces the old one
+    """
```

Comment:
You can shorten this description / straight to the point. E.g. "If an invite has expired and delete_invites cronjob has not yet run, the invite should be deleted and replace it with a new one".